### DISCUSSION TITLE
rocketmq grpc sdk

### DISCUSSION THREAD

**Discussion by weihubeats** [apache/rocketmq#6225] [Q&A] [answered]
rocketmq grpc access I saw the demo in rocketmq-clients, do we have a grpc access sdk that can be used directly by reference

**Comment by ShadowySpirits** [comment, marked as answer]
I don't understand what you mean. If you want to have the benefit of grpc protocol, you need to set up a proxy node and introduce one of [rocketmq-clients](https://github.com/apache/rocketmq-clients) in your app, see [java examples](https://github.com/apache/rocketmq-clients/tree/master/java/client/src/main/java/org/apache/rocketmq/client/java/example) for more details.
